types/filemaker-webviewer/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for non-npm package filemaker-webviewer 1.0
+// Project: https://claris.com
+// Definitions by: Eric Luce <https://github.com/eluce2>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export enum FMScriptOption {
```

Comment:
Declaring an enum as exported from a module is problematic for several reasons. First, because there is no module, so importing anything from `"filemaker-webviewer"` should never be legal. Second, it seems that there’s also no enum or enum-like object in the implementation; the `option` param just takes the string `"0" | "1" | "2" | "3" | "4" | "5"`.

Reply:
The enum resolves to those numbers and allows a more easy-to-read way to pass an option to that function. Would I be able to export an object as const instead?

I understand there is no module on NPM here, but there is extra JavaScript loaded onto the webpage that requires better typing. Developers who import these types will understand this special use case

Devs should to be able to call the function like:
```ts
PerformScriptWithOption("scriptName", "scriptParam", FMScriptOption.RESUME)
```

Reply:
They shouldn’t be able to call it like that because `FMScriptOption` will not exist at runtime. When you say `export enum` you are asserting that a real object exists in a real module called `"filemaker-webviewer"`.

Reply:
Ah, I see. I will remove from here then and add the enum to my helper package for these functions.